BungeeCord-Patches/0060-Waterdog-PE-modifications.patch

```diff
@@ -1351,6 +1446,7 @@ index 139c4f09..51ee3a8b 100644
 +                user.unsafe().sendPacket( new network.ycc.waterdog.pe.packet.PEStopSoundPacket( "portal.travel", true ) );
```

Comment:
Depending on delay this will sometimes crash the client^

Reply:
This was there before